meta-mentor-staging/recipes-graphics/mesa/mesa-demos_8.2.0.bbappend

```diff
@@ -0,0 +1,5 @@
+FILESEXTRAPATHS_prepend_amd := "${THISDIR}/${PN}_${PV}:"
+
+SRC_URI_append_amd = "\
+			file://0010-sharedtex_mt-fix-rendering-thread-hang.patch \
```

Comment:
If there's only one file, don't do a multiline value with line continuation line this.